Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageMetadataView.swift

```diff
@@ -8,7 +8,12 @@ import UIKit
 public typealias ChatMessageMetadataView = _ChatMessageMetadataView<NoExtraData>
 
 open class _ChatMessageMetadataView<ExtraData: ExtraDataTypes>: _View, UIConfigProvider {
```

Comment:
The view defines a small customization unit for a specific portion of content (metadata). Honestly, I like this being encapsulated in one view (I'd also do the same for thread view) 😄 I think having 2 layers is OK and it would also improve content view readability (the layout and content methods + API in general)